contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -438,7 +429,7 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
         for (uint256 i = 0; i < messageIDs.length; i++) {
             uint256 receivedMessageID = messageIDs[i];
             // Check the relayer reward address for this message.
```

Comment:
```suggestion
            // Check that the message has been received.
```

Reply:
I think "Get the relayer reward address for the message" is more clear. I'll update.